### PR TITLE
Add randomize lines example

### PR DESCRIPTION
**Pull request by ethanpil**

### CODE REVIEW

**Comment by dail8859** [comment]
Both this and #55 are useful scripts but since they are so similar I don't think there is any benefit including both as examples. Do you have a preference as to which one?

**Comment by ethanpil** [comment]
I guess the randomize lines might be slightly more useful to most people, although I often use both.

File: examples/randomizelines.lua [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---------------------------------------------------------------
+-- randomizelines.lua for Notepad++ LuaScript plugin
+--
+-- This script will randomize the lines within a document.
+--
+-- Activate it on all text in the editor with CTRL+F5
```
Comment (dail8859) [inline examples/randomizelines.lua]: What does `Ctrl+F5` do?

File: examples/randomizelines.lua [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+-- or select a group of lines to randomize only the selection.
+---------------------------------------------------------------
+
+-- helper function to split lines
+function split(str, sep)
```
Comment (dail8859) [inline examples/randomizelines.lua]: Not a big deal but this can be made a `local` function.

**Comment by ethanpil** [comment]
I made the change to the function to localize the scope.

**Comment by dail8859** [comment]
Thanks, can you also remove the mentioning of `Ctrl+F5` since I can't seem to figure out what it does?

**Comment by dail8859** [comment]
Thanks for the PR!

**Pull request merged**